Sources/Starscream/WebSocket.swift

```diff
@@ -85,6 +85,46 @@ public enum WebSocketEvent {
     case viabilityChanged(Bool)
     case reconnectSuggested(Bool)
     case cancelled
+    case peerClosed
+}
+
+public enum TLSVersion: Int {
+
+    case v1_0
+    case v1_1
+    case v1_2
+    case v1_3
```

Comment:
Should we remove TLSVersions that we don't support anymore out of security reasons? I think it's 1.0 and 1.1, but didn't validate it.

Reply:
as it's a dependency, and in case it's ported to main (still need to do the PR + get it merged - nothing got merged for a while) repo I would keep the versions in for now.